docs/user-guide/meta-models.md

```diff
@@ -381,6 +381,62 @@ The `OutlierClassifier` can be combined with any classification model in the `St
 
 --8<-- "docs/_static/meta-models/outlier-classifier-stacking.html"
 
+## Ordinal Classification
+
+Ordinal classification (sometimes also referred to as Ordinal Refression) involves predicting an ordinal target variable, where the classes have a meaningful order.
+Examples of this kind of problem are: predicting customer satisfaction on a scale from 1 to 5, predicting the severity of a disease, predicting the quality of a product, etc.
+
+The [`OrdinalClassifier`][ordinal-classifier-api] is a meta-model that can be used to transform any classifier into an ordinal classifier by fitting N-1 binary classifiers, each handling a specific class boundary, namely: $P(y <= 1), P(y <= 2), ..., P(y <= N-1)$.
+
```

Comment:
Bit of a nit, but might it make sense to add this diagram ...

![CleanShot 2024-01-26 at 15 57 39](https://github.com/koaning/scikit-lego/assets/1019791/b63c02b1-cf1c-4ff0-afc9-171c55626b97)

... somewhere over here? ... 

![CleanShot 2024-01-26 at 15 56 56](https://github.com/koaning/scikit-lego/assets/1019791/05af021b-98a0-49d6-a50d-44ccab0c4b11)

Reply:
Makes 110% sense! Collapsed admonition should do the trick